### PR TITLE
sentinel: tfplan, add destroy and requires_new keys

### PR DESCRIPTION
**Pull request by vancluever**
This documents the upcoming destroy and requires_new keys, which can
be used to determine if 1) a resource is being destroyed and 2) if a
new resource is required to satisfy a diff.

### CODE REVIEW

File: content/source/docs/cloud/sentinel/import/tfplan.html.md [hunk 1]
```diff
@@ -412,6 +414,63 @@ particular complex value within the resource.
...
+
+main = rule { tfplan.resources.null_resource.foo[0].destroy }
+```
+
+### Value: `requires_new`
+
+* **Value Type:** Boolean.
+
+The `requires_new` value within the [resource
+namespace](#namespace-resources-data-sources) is `true` if resource needs to be
```
Comment (nfagerlund) [inline content/source/docs/cloud/sentinel/import/tfplan.html.md]:
```suggestion
namespace](#namespace-resources-data-sources) is `true` if the resource is still present in the configuration, but must be
```

File: content/source/docs/cloud/sentinel/import/tfplan.html.md [hunk 2]
```diff
@@ -412,6 +414,63 @@ particular complex value within the resource.
...
+destroyed to satisfy its current diff. In this case, [`destroy`](#value-destroy)
```
Comment (nfagerlund) [inline content/source/docs/cloud/sentinel/import/tfplan.html.md]:
```suggestion
destroyed to satisfy its current diff. Whenever `requires_new` is `true`, [`destroy`](#value-destroy)
```

File: content/source/docs/cloud/sentinel/import/tfplan.html.md [hunk 3]
```diff
@@ -412,6 +414,63 @@ particular complex value within the resource.
...
+is also set.
```
Comment (nfagerlund) [inline content/source/docs/cloud/sentinel/import/tfplan.html.md]:
```suggestion
is also `true`.
```

**Comment by vancluever** [comment]
Thanks @thrashr888 and @nfagerlund!

**Pull request merged**